common_repo_templates/CONTRIBUTING_template.md

```diff
@@ -16,18 +17,22 @@ All contributions to the repository must be submitted under the terms of the [Ap
 
 ## Certificate of Origin
 
-By contributing to this project, you agree to the Developer Certificate of Origin (DCO). This document was created by the Linux Kernel community and is a simple statement that you, as a contributor, have the legal right to make the contribution. See the [DCO](DCO) file for details.
+By contributing to this project, you agree to the Developer Certificate of Origin (DCO). This document was created by the Linux Kernel community and is a simple statement that you, as a contributor, have the legal right to make the contribution. See the [DCO](https://github.com/open-cluster-management/community/blob/main/DCO) file for details.
 
 ## DCO Sign Off
 
-You must sign your commit to state that you certify the [DCO](DCO). To sign your commit, add a line like the following at the end of your commit message:
+You must sign off your commit to state that you certify the [DCO](https://github.com/open-cluster-management/community/blob/main/DCO). To certify your commit for DCO, add a line like the following at the end of your commit message:
```

Comment:
for the link change I don't think its necessary because each repo will still have their own DCO file as the community convention but this can work too.